plugins/inputs/cisco_telemetry_mdt/README.md

```diff
@@ -27,6 +27,12 @@ later.
  ## Grpc Maximum Message Size, default is 4MB, increase the size.
  max_msg_size = 4000000
 
+ ## GRPC permit keepalives without calls, set to true if your clients are sending pings without calls in-flight.
+ # permit_keepalive_without_calls = false
```

Comment:
How does a user know if their clients are sending pings without calls in flight? Basically, if I am using Telegraf, what would I need to know to change this from the default? Similar to what you added for the keepalive_minimum_time below description, you specified an error message that helps guide the user.

@reimda thoughts on a different variable to avoid a boolean?

Reply:
Thanks - I'd still like @reimda's thoughts on a way to avoid the boolean or to go with it given it is directly used to configure the connection.

Reply:
That sounds good to me - after all, these settings are only useful to get around implementation issues, and don't have any effect on the TCP transport anyway. I'm looking at the opcua plugin to see how the workaround section is implemented, I could modify this to have a "grpc_enforcement_policy" section with these settings.